dvc/stage.py

```diff
@@ -702,7 +702,11 @@ def run(self, dry=False, resume=False, no_commit=False):
         else:
             logger.info("Running command:\n\t{}".format(self.cmd))
             if not dry:
-                if not self.is_callback and self._already_cached():
+                if (
+                    not force
```

Comment:
Have you seen comment about import from https://github.com/iterative/dvc/issues/1667#issuecomment-468229262 ? Should we take --force into account for import stages as well or am I missing something?

Reply:
Sorry, @efiop, I didn't get it right away; why should we need to incorporate `--force` to import if it is intended to run it "once", if you want to re-download it, why wouldn't you `dvc repro --force` the stage file?

Reply:
Here is an example:
```
#!/bin/bash               
                          
set -e                    
set -x                    
                          
rm -rf myrepo             
mkdir myrepo              
cd myrepo                 
                          
git init                  
dvc init                  
                          
echo foo > foo            
dvc import foo bar        
                          
dvc repro bar.dvc         
                          
dvc repro bar.dvc --force 
```

which will print

```
+ dvc repro bar.dvc                                                                                                        
Stage 'bar.dvc' didn't change.                                                                                             
Pipeline is up to date. Nothing to reproduce.                                                                              
+ dvc repro bar.dvc --force                                                                                                
Stage 'bar.dvc' didn't change.                                                                                             
Reproducing 'bar.dvc'                                                                                                      
Importing '/home/efiop/git/dvc/myrepo/foo' -> '/home/efiop/git/dvc/myrepo/bar'                                             
Checking out '{'scheme': 'local', 'path': '/home/efiop/git/dvc/myrepo/bar'}' with cache 'd3b07384d113edec49eaa6238ad5ff00'.
Output 'bar' didn't change. Skipping saving.                                                                               
Saving 'bar' to cache '.dvc/cache'.                                                                                        
Saving information to 'bar.dvc'.                                                                                           
                                                                                                                           
To track the changes with git run:                                                                                         
                                                                                                                           
        git add bar.dvc                                                                                                    
```

as you can see `dvc repro bar.dvc --force` just _checks out_ bar instead of actually re-downloading it.